plugins/OS2LAna.cc

```diff
@@ -720,19 +722,27 @@ bool OS2LAna::filter(edm::Event& evt, const edm::EventSetup& iSetup) {
   h1_["sqrtChi2"] ->Fill(sqrt(chi2_result.first), evtwt);
   h1_["chi2_mass"] ->Fill(chi2_result.second, evtwt);
 */
+  std::auto_ptr<double> ptr_fevtwt ( new double(evtwt) ) ;
+  std::auto_ptr<double> ptr_evtwt ( new double(presel_wt) ) ; 
+  evt.put(ptr_fevtwt, "finalWeight");
+  evt.put(ptr_evtwt, "PreWeight");
+
```

Comment:
I don't need it for the skim. I want it for when I'm running on skims to store the weight for a later module.

Just like eventcleaner stores the weight to be read in os2lana